arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -845,116 +811,7 @@ kwiver::vital::image_container_sptr
 ffmpeg_video_input
 ::frame_image( )
 {
-  // Quick return if the stream isn't valid
```

Comment:
I'm not sure what all these major FFMPEG changes are doing, but I would prefer these be reviewed as a separate branch.  FFMPEG is a key functionality that is tricky to get right.  Changes could introduce errors across many projects.  It would be better to isolate these changes for separate review and explain the motivation behind them.

Reply:
#927 has been added with the isolated changes

Reply:
@mleotta  As were are addressing the concerns FFMPEG and now ocv image container changes in #927 are there an other concerns about this branch?